File: genealogy-tree-API/GenealogyTree.Domain/DTO/EducationModel.cs
using GenealogyTree.Domain.DTO.Generic;
using System;

namespace GenealogyTree.Domain.DTO
{
    public class EducationModel
    {
        public int Id { get; set; }
        public Guid UserId { get; set; }
        public string EducationInstituteName { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }

        public GenericNameModel EducationLevel { get; set; }
    }
}

File: gene-tree/src/app/modules/user/education/models/education.model.ts
import { Guid } from 'guid-typescript';
import { CommonObject } from 'src/app/modules/shared/models/common-object';
import { EducationLevelEnum } from "./education-level.enum";

export class EducationModel {
  constructor(){
    this.educationLevel = new CommonObject();
  }
  id: number;
  userId: Guid;
  educationInstituteName: string;
  educationLevel: CommonObject;
  startDate?: Date;
  endDate?: Date;
}
